server/dash/dashboards/overviewComb.py

```diff
@@ -0,0 +1,178 @@
+import textwrap
+import datetime
+import pandas as pd
+import plotly.express as px
+import plotly.graph_objects as go
+from dash import callback, dcc, html
+from dash.dependencies import Input, Output
+
+from config import API_HOST
+from design import CONFIG_OPTIONS, DISCRETE_COLORS, DISCRETE_COLORS_MAP, LABELS, apply_figure_style
+
+title = "OVERVIEW COMBINED DASHBOARD"
+
+# DATA
+print(" * Downloading data for dataframe")
+query_string = '/reports?field=type_name&field=council_name&field=created_date'
+df = pd.read_json(API_HOST + query_string)
+print(" * Dataframe has been loaded")
+
+# Loading the dataframe for the NCs and correspoding requests
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=council_name&filter=created_date>=2016-01-01"
+df1 = pd.read_json(API_HOST + query_string)
+df1 = df1.groupby(['council_name'])['counts'].sum().sort_values().to_frame()
+
+# Loading the data for the number of new requests
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=created_year&filter=created_date>=2016-01-01"
+df2 = pd.read_json(API_HOST + query_string)
+df2 = df2.groupby(['created_year'])['counts'].sum().to_frame()
+
+# Loading the count of each request types overall
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=type_name&filter=created_date>=2016-01-01"
+df3 = pd.read_json(API_HOST + query_string)
+df3 = df3.groupby(['type_name'], as_index=False)['counts'].sum()
+
+# Loading the total number of request source
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=source_name&filter=created_date>=2016-01-01"
+df6 = pd.read_json(API_HOST + query_string)
+reqSourceLab = df6.groupby(['source_name'])['counts'].sum()
+
+# Loading the number of Request Agencies
+query_string = "/reports?field=agency_name&filter=created_date>=2016-01-01"
+df5 = pd.read_json(API_HOST + query_string)
+reqAgency = df5.groupby(['agency_name'])['counts'].sum()
+
+# Request Share by Agency Pie Chart
+print(" * Downloading data for dataframe")
+query_string = "/reports?field=agency_name&filter=created_date>=2016-01-01"
+df5 = pd.read_json(API_HOST + query_string)
+df5 = df5.groupby(['agency_name'])['counts'].sum().to_frame()
+df5.sort_values('counts', ascending=False, inplace=True)
+df5.loc['Others'] = df5[4:].sum()
+df5.sort_values('counts', ascending=False, inplace=True)
+df5 = df5[:5]
+df5.index = df5.index.map(lambda x: '<br>'.join(textwrap.wrap(x, width=16)))
+
+shareReqByAgencyPieChart = px.pie(
```

Comment:
Python uses `snake_case`, not `camelCase` (except for class names), so let's use that here.

Reply:
Updated variable names to snake_case! Thanks for the catch!